systemd/ubuntu-advantage.service

```diff
@@ -0,0 +1,24 @@
+# If you are uninterested in the (free for personal use) Ubuntu Advantage
+# services, including security updates after standard EOL and kernel patching
+# without rebooting, then you can safely stop and disable this service:
+# sudo systemctl stop ubuntu-advantage.service
+# sudo systemctl disable ubuntu-advantage.service
```

Comment:
Should we add any considerations that this service only runs on GCP instances ?

Reply:
Ah yes I'll add that!

Reply:
Updated! Let me know if you'd prefer different wording